src/test/java/org/cactoos/func/CallableOfTest.java

```diff
@@ -47,4 +50,39 @@ public void convertsFuncIntoCallable() throws Exception {
         );
     }
 
+    @Test
+    public void convertsRunnableIntoCallable() throws Exception {
+        final AtomicBoolean flag = new AtomicBoolean(false);
+        new CallableOf<>(
+            (Runnable) () -> { flag.set(true); }
+        ).call();
+        MatcherAssert.assertThat(
+            flag.get(),
+            Matchers.is(true)
+        );
+    }
+
+    @Test
+    public void convertsProcIntoCallable() throws Exception {
+        final AtomicBoolean flag = new AtomicBoolean(false);
+        new CallableOf<>(
+            (Proc<AtomicBoolean>) (unused) -> { flag.set(true); }
+        ).call();
+        MatcherAssert.assertThat(
+            flag.get(),
+            Matchers.is(true)
+        );
+    }
+
+    @Test
+    public void convertsFuncWithInputIntoCallable() throws Exception {
+        MatcherAssert.assertThat(
+            new CallableOf<>(
+                (Func<Integer, Integer>) num -> num + 1,
```

Comment:
@llorllale Can you add accolades here? ``num -> { num + 1; }`` so we are consistent in all tests.

Reply:
@amihaiemil adding the brackets there will then force the expression to be changed to `num -> { return num + 1; }`. Is that what you want?

Reply:
@llorllale Right, then leave it like that. My bad, sorry, still hating the Java 8 syntax :(